com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -218,6 +220,49 @@ internal struct AgentParameters
         [FormerlySerializedAs("maxStep")]
         [HideInInspector] public int MaxStep;
 
+        /// <summary>
+        /// Options for controlling how the Agent class is searched for <see cref="ObservableAttribute"/>s.
+        /// </summary>
+        public enum ObservableAttributeHandling
```

Comment:
I don't like any of the names here :/ Suggestions welcome.

Reply:
ObservableAttributeOptions?

Reply:
I wonder if `SensorAttribute` makes it more consistent?

Reply:
changed to ObservableAttributeOptions